src/esp/nav/PathFinder.cpp

```diff
@@ -229,6 +231,8 @@ struct PathFinder::Impl {
 
   std::pair<vec3f, vec3f> bounds() const { return bounds_; };
 
+  std::vector<std::vector<bool>> getTopDownView(const float res);
```

Comment:
This should be an `Eigen::Matrix<bool, Eigen::Dynamic, Eigen::Dynamic>` matrix -- that'll get sent over to python as a numpy array with zero copy :)